.travis.yml

```diff
@@ -20,6 +20,7 @@ cache:
 
 before_script:
   - phpenv config-add travis.php.ini
+  - composer self-update 1.10.16
```

Comment:
Maybe it's better to use: `composer self-update --1`

Reply:
Superb advice!